internal/view/issues.go

```diff
@@ -113,15 +119,34 @@ func (s *issuesComponent) String() string {
 	return s.str
 }
 
+func joinIntroducedBy(elems []IntroducedBy) string {
```

Comment:
I'm fine with calling it joinIntroducedBy for now, then let's see what prefix to use. It's not really a "join" as it is right now :)